Rocket.Chat/Controllers/Auth/ConnectServerViewController.swift

```diff
@@ -78,7 +99,29 @@ final class ConnectServerViewController: BaseViewController {
             textFieldServerURL.text = applicationServerURL.host
             connect()
         }
+        NotificationCenter.default.addObserver(self, selector: #selector(internetConnection), name: Notification.Name.reachabilityChanged, object: reachability)
+                do {
+                        try reachability.startNotifier()
+                    } catch {
+                            print(error)
+                        }
+        
+        
     }
+    override func viewWillDisappear(_ animated: Bool) {
+        super.viewDidDisappear(true)
+                    NotificationCenter.default.removeObserver(self)
+    }
+    @objc func internetConnection(notification: NSNotification) {
```

Comment:
Is this needed? It only prints to console connection status

Reply:
Its not needed , I used it for the testing .